packages/babel-helpers/src/helpers.js

```diff
@@ -500,11 +500,23 @@ helpers.objectWithoutProperties = defineHelper(`
   }
 `);
 
+helpers.assertThisInitialized = defineHelper(`
+  export default function _assertThisInitialized(self) {
+    if (!self) {
```

Comment:
I would prefer to be more explicit in the test here, like:  `typeof self === 'undefined'`. It's faster.

Reply:
`typeof self === "undefined"` or `self === void 0`?

Reply:
I set up a not too much significative jsperf (https://jsperf.com/x-vs-typeof-vs-void-0/1) and they are almost the same. I'll stick with the second one since it is shorter.

Reply:
Great, thanks!